Library/Formula/fish.rb

```diff
@@ -15,26 +15,19 @@ class Fish < Formula
     url "https://github.com/fish-shell/fish-shell.git", :shallow => false
 
     depends_on "autoconf" => :build
-    # Indeed, the head build always builds documentation
     depends_on "doxygen" => :build
   end
 
-  skip_clean "share/doc"
-
   def install
     system "autoconf" if build.head?
     # In Homebrew's 'superenv' sed's path will be incompatible, so
     # the correct path is passed into configure here.
     system "./configure", "--prefix=#{prefix}", "SED=/usr/bin/sed"
     system "make", "install"
-  end
 
-  def post_install
-    system "pkill fishd || true"
```

Comment:
Is there absolutely no other way to get around the potential problem? Killing users' running processes rubs up against Homebrew's philosophy a little.